frontend/src/screens/ConnectAlbyAccount.tsx

```diff
@@ -47,10 +47,19 @@ export function ConnectAlbyAccount({ connectUrl }: ConnectAlbyAccountProps) {
               </CardDescription>
             </CardHeader>
           </Card>
+          <Card className="w-full">
+            <CardHeader className="flex flex-col justify-center items-center text-center p-4">
+              <DatabaseBackup className="w-6 h-6" />
+              <CardTitle className="text-sm">Encrypted Backups</CardTitle>
+              <CardDescription className="text-xs">
+                Ensures you can always recover funds from lightning channels
+              </CardDescription>
+            </CardHeader>
+          </Card>
           <Card className="w-full">
             <CardHeader className="flex flex-col justify-center items-center text-center p-4">
               <LifeBuoy className="w-6 h-6" />
-              <CardTitle className="text-sm">Support</CardTitle>
+              <CardTitle className="text-sm">Real Support</CardTitle>
```

Comment:
Not sure about the word "real" here. is the other one "fake"? :)

Reply:
True, I was thinking about chatbots. But I will undo this change.